Sources/Sentry/SentryClient.m

```diff
@@ -168,20 +176,20 @@ - (SentryId *)captureError:(NSError *)error
 
 - (SentryId *)captureError:(NSError *)error withScope:(SentryScope *)scope
 {
-    SentryEvent *event = [self buildErrorEvent:error];
+    SentryEvent *event = [self buildErrorEvent:error scope:scope];
     return [self sendEvent:event withScope:scope alwaysAttachStacktrace:YES];
 }
 
 - (SentryId *)captureError:(NSError *)error
                withSession:(SentrySession *)session
                  withScope:(SentryScope *)scope
 {
-    SentryEvent *event = [self buildErrorEvent:error];
+    SentryEvent *event = [self buildErrorEvent:error scope:scope];
     event = [self prepareEvent:event withScope:scope alwaysAttachStacktrace:YES];
     return [self sendEvent:event withSession:session withScope:scope];
 }
 
-- (SentryEvent *)buildErrorEvent:(NSError *)error
+- (SentryEvent *)buildErrorEvent:(NSError *)error scope:(SentryScope *)scope
```

Comment:
`m`: I think the scope is not used buildErrorEvent
```suggestion
- (SentryEvent *)buildErrorEvent:(NSError *)error
```

Reply:
You're right. After the decoupling we don't need the scope in this function.